subsys/mpsl/init/Kconfig

```diff
@@ -74,6 +74,29 @@ config MPSL_LOW_PRIO_IRQN
 	  This option sets the low priority interrupt that MPSL will use.
 	  Check the interrupt number in the MDK
 
+config MPSL_HFCLK_LATENCY
+	int
+	default 850 if SOC_SERIES_NRF54HX
```

Comment:
850 and 856 is for the nRF54H-DK / nRF54L-DK/PDK ? Maybe we should not set it default so low for ALL nRF54 boards ?

Reply:
850 is taken from the clock driver we'll use for 54H family products https://github.com/zephyrproject-rtos/zephyr/pull/77365/files#diff-9c26040f4e2aaad162832d41fc1a1f35ca511eb33b3e0a4f94d3de92b9d86395R14

856 is the value we already use on 54L, there's no actual change on this PR, just documentation https://projecttools.nordicsemi.no/bitbucket/projects/DRGN/repos/mpsl/browse/libs/scheduling/src/mpsl_clock.c#251

Reply:
although I see now the code actually uses 854, not 856. But it doesn't really matter

Reply:
now it's 854 on both. simpler